core/spec.md

```diff
@@ -2248,37 +2421,25 @@ Where:
 - if an `epoch` value is specified then the server MUST check to ensure that
   the value matches the current `epoch` value and if it differs then an error
   MUST be generated
-- if `RESOURCEUri` is present then the body MUST be empty and any
+- if `RESOURCEUrl` is present then the body MUST be empty and any
   local Resource contents MUST be erased
 - if `format` is present and changing it would result in Versions of this
   Resource becoming invalid with respect to their own `format` values, then
   an error MUST be generated
-- if the body is empty and `RESOURCEUri` is absent then the Resource's
+- if the body is empty and `RESOURCEUrl` is absent then the Resource's
   contents MUST be erased
 - a request to update a read-only attribute MUST be silently ignored
-- a request to update a mutable attribute with an invalid value MUST generate
-  an error
+- a request to update a mutable attribute with an invalid value MUST
+  generate an error (this includes deleting a mandatory mutable attribute)
 - complex attributes that have nested values (eg. `tags`) MUST be specified
   in their entirety
-- a request that is missing a mandatory attribute MUST generate an error.
-  If a manadatory attribute is too large for an HTTP header value then
-  this operation MUST be done via a PUT to the `?meta` API
 
-Missing Registry HTTP headers MUST NOT be interpreted as a request to delete
-the attribute as it is impossible to know if it is missing due to a desire to
-delete it or if the value is too large to be serialized as an HTTP header.
-In this respect, processing of the HTTP headers is similar to how an HTTP
-`PATCH` behaves - meaning, only the attributes wishing to be updated are
-included in the request.
+Missing Registry HTTP headers MUST be interpreted as a request to delete the
```

Comment:
I see that causing a lot of inadvertent data loss. If you want to override a value, you need to include the header, IMO.